packages/docs-gesture-handler/docs/gestures/_shared/base-gesture-event-data.md

```diff
@@ -1,19 +1,27 @@
 ### Event attributes common to all gestures:
```

Comment:
Similarly to config and callbacks, I think we can do without the subheaders for events.